extensions/devservices/common/src/main/java/io/quarkus/devservices/common/ContainerAddress.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.devservices.common;
+
+public class ContainerAddress {
+    private final String ipAddress;
+    private final int port;
+
+    public ContainerAddress(String ipAddress, Integer port) {
```

Comment:
can the port be `null`? If not, why `Integer` and not `int`?

Reply:
Theoretically, it can, as the [ContainerPort#getPublicPort](https://github.com/docker-java/docker-java/blob/master/docker-java-api/src/main/java/com/github/dockerjava/api/model/ContainerPort.java#L67) is annoted with `CheckForNull`.

Reply:
Correct, so I added an explicit check for this unlikely event, to be on the safe side.